tests/robottelo/test_orm.py

```diff
@@ -115,8 +113,6 @@ def setUp(self):  # noqa pylint:disable=C0103
 
     def tearDown(self):  # noqa pylint:disable=C0103
         """Restore backed-up objects."""
-        client.delete = self.client_delete
-        client.get = self.client_get
```

Comment:
All three tests in this class now mock `client.get` and `client.delete` using context managers, so this is no longer necessary.